README.md

```diff
@@ -1 +1,59 @@
-# To-Do
\ No newline at end of file
+# To-Do
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This a group work project testing basic Javascript skills.
+
+# WORKFLOW SCREENSHOT
+![screenshot]()
```

Comment:
- [ ] It seems you forgot to edit the standard empty README file. Please make sure to do so and add a valid image of the app.